### PR TITLE
Adding environments documentation

### PR DESCRIPTION
**Pull request by notABluesSinger**

### CODE REVIEW

**Comment by lukefrake** [comment]
+1

File: environments.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Environments
+We should provide three environments to suit our deployment and testing process
+
+### Development
+When working locally and developing.
+This should point to local copies of all conected systems
+
+### Staging
+That final step, where the work can be tested before releasing to the wild.
+This should point to staging versions of all conected systems
```
Comment (robhuzzey) [inline environments.md]: Typo "connected"

File: environments.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+### Production
+The real deal, everything live and in front of customers.
+This should point to other productions systems
+
+###Environment Switching
+Each system should support the ability to select the environment it is running in, in the following order
+
+- Command line argument, when starting the application
```
Comment (robhuzzey) [inline environments.md]: Are we using argv module (like the wall) for this... is there a standard? ie: --environment=live

Comment (notABluesSinger) [inline environments.md]: I don't want to prescribe how we achieve this here. This documentation is platform agnostic, so could be applied to any other language

**Comment by robhuzzey** [comment]
+1

**Pull request merged**